actors/miner/tests/miner_actor_test_commitment.rs

```diff
@@ -502,7 +505,8 @@ mod miner_actor_test_commitment {
                 ),
             );
             rt.reset();
-            precommit_params.seal_proof = RegisteredSealProof::StackedDRG32GiBV1P1;
+            precommit_params.seal_proof =
+                RegisteredSealProof::StackedDRG32GiBV1P1_Feat_SyntheticPoRep;
```

Comment:
I don't think we want to change any existing tests here. Possibly we need some new tests, but if this value is just plumbed directly to the syscall, we're probably ok without.

Reply:
Thank you! If we do end up needing a separate test, let's figure out the best way to do that.